samples/rag-aisearch/README.md

```diff
@@ -0,0 +1,71 @@
+# Semantic Search Demo
+
+## Prerequisites
+
+The sample is available in the following language stacks:
+
+* [C# on the in process worker](csharp-inproc/)
+
+Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
+
+## Running the sample
+
+This sample requires creating an Azure AI Search with Semantic Ranker. You can do this by following the [Azure AI Search quickstart](https://learn.microsoft.com/en-us/azure/search/search-create-service-portal)
+and optionally [enable semantic ranking](https://learn.microsoft.com/en-us/azure/search/semantic-how-to-enable-disable?tabs=enable-portal) for semantic ranking and captions features.
+
+Once you have an Azure AI Search resource, you can run the sample by following these steps:
+
+1. Update the `AISearchEndpoint` value in `local.settings.json` to match your Azure AI Search endpoint.
+1. Provide SearchAPIKey for API key based auth, skip setting it for managed identity based authentication.
+1. In case of multiple search providers, it is necessary to configure the type from the `host.json` as shown in below snippet.
+1. Use of Semantic Search, Semantic Captions and Vector Search Dimensions are configurable. You may configure the `host.json` file within the project and following example shows the default values:
+
+    ```json
+    "extensions": {
+        "openai": {
+            "searchProvider": {
+                "type": "azureaisearch",
+                "isSemanticSearchEnabled": "true",
+                "useSemanticCaptions": "true",
+                "vectorSearchDimensions": "1536"
+            }
+        }
+    }
+    ```
+
+    `VectorSearchDimensions` is length of the embedding vector. The dimensions attribute has a minimum of 2 and a maximum of 3072 floating point values each. By default, the length of the embedding vector will be 1536 for text-embedding-3-small.
+
+1. Use a terminal window to navigate to the sample directory
+
+    ```sh
+    cd samples/rag-aisearch/csharp-inproc
+    ```
+
+1. Build and start the app
+
+    ```sh
+    dotnet build && cd bin/debug/net6.0 && func start
```

Comment:
Actually, I think we can just do `func start`. That should also take care of the build step.